doc/api/source/index.html.md.erb

```diff
@@ -77,9 +110,10 @@ curl https://www.gov.uk/api/content/vat-rates | python -m json.tool # Pretty pri
 
 The value of `content_id` is the value you need to pass to this API.
```

Comment:
Pass the `content_id` value to the content performance API.

Reply:
Done